ChangeLog.d/MD-X.509-hashing.txt

```diff
@@ -0,0 +1,2 @@
+Features
+   * x.509 now uses PSA for hashing (conditionally on MBEDTLS_USE_PSA_CRYPTO).
```

Comment:
Missing newline at the end of this file. Some of our scripts care about this (see CI).